config/software/ncurses.rb

```diff
@@ -51,27 +51,27 @@
     # These patches are taken from NetBSD pkgsrc and provide GCC 4.7.0
     # compatibility:
     # http://ftp.netbsd.org/pub/pkgsrc/current/pkgsrc/devel/ncurses/patches/
-    patch source: "patch-aa", plevel: 0
-    patch source: "patch-ab", plevel: 0
-    patch source: "patch-ac", plevel: 0
-    patch source: "patch-ad", plevel: 0
-    patch source: "patch-cxx_cursesf.h", plevel: 0
-    patch source: "patch-cxx_cursesm.h", plevel: 0
+    patch source: "patch-aa", plevel: 0, env: env
+    patch source: "patch-ab", plevel: 0, env: env
+    patch source: "patch-ac", plevel: 0, env: env
+    patch source: "patch-ad", plevel: 0, env: env
+    patch source: "patch-cxx_cursesf.h", plevel: 0, env: env
+    patch source: "patch-cxx_cursesm.h", plevel: 0, env: env
 
     # Opscode patches - <someara@opscode.com>
     # The configure script from the pristine tarball detects xopen_source_extended incorrectly.
     # Manually working around a false positive.
-    patch source: "ncurses-5.9-solaris-xopen_source_extended-detection.patch", plevel: 0
+    patch source: "ncurses-5.9-solaris-xopen_source_extended-detection.patch", plevel: 0, env: env
   end
 
   # AIX's old version of patch doesn't like the patches here
   unless aix?
     if version == "5.9"
       # Update config.guess to support platforms made after 2010 (like aarch64)
-      patch source: "config_guess_2015-09-24.patch", plevel: 0
+      patch source: "config_guess_2015-09-24.patch", plevel: 0, env: env
```

Comment:
yeah, i'm not sure why you're editing these patches that only go against AIX, but the fact that you're touching every single patch statement is screaming out to me to just change omnibus itself.